common/shlibs

```diff
@@ -3259,110 +3259,148 @@ libgnuradio-pdu.so.3.10.3 gnuradio-3.10.3.0_1
+libvtkFiltersPoints-9.2.so.1 vtk-9.2.2_1
+libvtkFiltersProgrammable-9.2.so.1 vtk-9.2.2_1
+libvtkFiltersSelection-9.2.so.1 vtk-9.2.2_1
+libvtkFiltersSMP-9.2.so.1 vtk-9.2.2_1
+libvtkFiltersSources-9.2.so.1 vtk-9.2.2_1
+libvtkFiltersStatistics-9.2.so.1 vtk-9.2.2_1
+libvtkFiltersTexture-9.2.so.1 vtk-9.2.2_1
+libvtkFiltersTopology-9.2.so.1 vtk-9.2.2_1
+libvtkFiltersVerdict-9.2.so.1 vtk-9.2.2_1
+libvtkfmt-9.2.so.1 vtk-9.2.2_1
+libvtkfreetype-9.2.so.1 vtk-9.2.2_1
+libvtkGeovisCore-9.2.so.1 vtk-9.2.2_1
+libvtkgl2ps-9.2.so.1 vtk-9.2.2_1
+libvtkglew-9.2.so.1 vtk-9.2.2_1
+libvtkhdf5_hl-9.2.so.1 vtk-9.2.2_1
+libvtkhdf5-9.2.so.1 vtk-9.2.2_1
+libvtkImagingColor-9.2.so.1 vtk-9.2.2_1
+libvtkImagingCore-9.2.so.1 vtk-9.2.2_1
+libvtkImagingFourier-9.2.so.1 vtk-9.2.2_1
+libvtkImagingGeneral-9.2.so.1 vtk-9.2.2_1
+libvtkImagingHybrid-9.2.so.1 vtk-9.2.2_1
+libvtkImagingMath-9.2.so.1 vtk-9.2.2_1
+libvtkImagingMorphological-9.2.so.1 vtk-9.2.2_1
+libvtkImagingSources-9.2.so.1 vtk-9.2.2_1
+libvtkImagingStatistics-9.2.so.1 vtk-9.2.2_1
+libvtkImagingStencil-9.2.so.1 vtk-9.2.2_1
+libvtkInfovisCore-9.2.so.1 vtk-9.2.2_1
+libvtkInfovisLayout-9.2.so.1 vtk-9.2.2_1
+libvtkInteractionImage-9.2.so.1 vtk-9.2.2_1
+libvtkInteractionStyle-9.2.so.1 vtk-9.2.2_1
+libvtkInteractionWidgets-9.2.so.1 vtk-9.2.2_1
+libvtkIOAMR-9.2.so.1 vtk-9.2.2_1
+libvtkIOAsynchronous-9.2.so.1 vtk-9.2.2_1
+libvtkIOCesium3DTiles-9.2.so.1 vtk-9.2.2_1
+libvtkIOCGNSReader-9.2.so.1 vtk-9.2.2_1
+libvtkIOChemistry-9.2.so.1 vtk-9.2.2_1
+libvtkIOCityGML-9.2.so.1 vtk-9.2.2_1
+libvtkIOCONVERGECFD-9.2.so.1 vtk-9.2.2_1
+libvtkIOCore-9.2.so.1 vtk-9.2.2_1
+libvtkIOEnSight-9.2.so.1 vtk-9.2.2_1
+libvtkIOExodus-9.2.so.1 vtk-9.2.2_1
+libvtkIOExport-9.2.so.1 vtk-9.2.2_1
+libvtkIOExportGL2PS-9.2.so.1 vtk-9.2.2_1
+libvtkIOExportPDF-9.2.so.1 vtk-9.2.2_1
+libvtkIOGeometry-9.2.so.1 vtk-9.2.2_1
+libvtkIOHDF-9.2.so.1 vtk-9.2.2_1
+libvtkIOImage-9.2.so.1 vtk-9.2.2_1
+libvtkIOImport-9.2.so.1 vtk-9.2.2_1
+libvtkIOInfovis-9.2.so.1 vtk-9.2.2_1
+libvtkIOIOSS-9.2.so.1 vtk-9.2.2_1
+libvtkIOLegacy-9.2.so.1 vtk-9.2.2_1
+libvtkIOLSDyna-9.2.so.1 vtk-9.2.2_1
+libvtkIOMINC-9.2.so.1 vtk-9.2.2_1
+libvtkIOMotionFX-9.2.so.1 vtk-9.2.2_1
+libvtkIOMovie-9.2.so.1 vtk-9.2.2_1
+libvtkIONetCDF-9.2.so.1 vtk-9.2.2_1
+libvtkIOOggTheora-9.2.so.1 vtk-9.2.2_1
+libvtkIOParallel-9.2.so.1 vtk-9.2.2_1
+libvtkIOParallelXML-9.2.so.1 vtk-9.2.2_1
+libvtkIOPLY-9.2.so.1 vtk-9.2.2_1
+libvtkIOSegY-9.2.so.1 vtk-9.2.2_1
+libvtkIOSQL-9.2.so.1 vtk-9.2.2_1
+libvtkioss-9.2.so.1 vtk-9.2.2_1
+libvtkIOTecplotTable-9.2.so.1 vtk-9.2.2_1
+libvtkIOVeraOut-9.2.so.1 vtk-9.2.2_1
+libvtkIOVideo-9.2.so.1 vtk-9.2.2_1
+libvtkIOXML-9.2.so.1 vtk-9.2.2_1
+libvtkIOXMLParser-9.2.so.1 vtk-9.2.2_1
+libvtkjpeg-9.2.so.1 vtk-9.2.2_1
+libvtkjsoncpp-9.2.so.1 vtk-9.2.2_1
+libvtkkissfft-9.2.so.1 vtk-9.2.2_1
+libvtklibharu-9.2.so.1 vtk-9.2.2_1
+libvtklibproj-9.2.so.1 vtk-9.2.2_1
+libvtklibxml2-9.2.so.1 vtk-9.2.2_1
+libvtkloguru-9.2.so.1 vtk-9.2.2_1
+libvtklz4-9.2.so.1 vtk-9.2.2_1
```

Comment:
Is this required by some package? If not, we should not include it in shlibs and keep this as small as possible.

Reply:
No. I had included only the ones that xbps-src requires at first, added these based on https://github.com/void-linux/void-packages/pull/40423#discussion_r1021732889.

Reply:
ok, that's wrong, please revert it

Reply:
done